### ISSUE TITLE
[MU4 Issue] Accessibility: unable to mutli-select instruments in Parts dialog

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#11848]
**Describe the bug**
I can't find a way to select multiple instruments in the Parts dialog using keyboard alone.  None of the strandard things I tried for mutli-select work.

I see the Parts dialog is in for some updates, and realistically, the mere addition of an Open All button as proposed in https://github.com/musescore/MuseScore/issues/11420 will go a long ways toward addressing this, as at least that common operation will become simple.  Still, ability to select individual parts to open is important also.  I've posted this as a separate issue to make sure that this gets looked at independently of whether the dialog is redesigned or not.

**To Reproduce**
1. Create or open a score for multiple instruments
2. Go to Parts dialog
3. Attempt to select multiple instruments using keyboard alone
5. See error - no response from Ctrl+A, Space, Ctrl+Space, Shift+Up/Down, etc

**Expected behavior**
Ctrl+A should select all
Shift+Up/Down should extend the selection
EIther Space, Shift+Space, or Ctrl+Space should toggle selection of current element

Basically, it should behave like a standard list widget that supports multiselect

**Screenshots**

**Platform information**
 - Tested on Linux

**Comment by ghost** [comment]
Same in Parts dialog (https://github.com/musescore/MuseScore/issues/18456) New Score Wizard/Instruments dialog (https://github.com/musescore/MuseScore/issues/29376), at least with KDE/Wayland.
